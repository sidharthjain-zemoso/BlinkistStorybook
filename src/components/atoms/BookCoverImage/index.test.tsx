import { render, screen } from "@testing-library/react";
import CoverImage from "./";

test("renders cover image component", () => {
  render(<CoverImage url="/res/cover images/2.png" />);

  const imageElement = screen.getByRole("img");
  expect(imageElement).toMatchSnapshot();
});
