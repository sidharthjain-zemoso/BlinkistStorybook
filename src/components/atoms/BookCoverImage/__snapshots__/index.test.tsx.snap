// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders cover image component 1`] = `
<img
  alt="book cover"
  src="/res/cover images/2.png"
/>
`;
